src/main/java/com/coreos/jetcd/KV.java

```diff
@@ -20,36 +20,72 @@
 @Beta
 public interface KV {
 
-  // ***************
-  // Op.PUT
-  // ***************
-
+  /**
+   * put a key-value pair into etcd.
+   *
+   * @param key key in ByteSequence
+   * @param value value in ByteSequence
+   * @return PutResponse
+   */
   CompletableFuture<PutResponse> put(ByteSequence key, ByteSequence value);
 
+  /**
+   * put a key-value pair into etcd.
+   *
+   * @param key key in ByteSequence
+   * @param value value in ByteSequence
+   * @param option PutOption
+   * @return PutResponse
+   */
   CompletableFuture<PutResponse> put(ByteSequence key, ByteSequence value, PutOption option);
 
-  // ***************
-  // Op.GET
-  // ***************
-
+  /**
+   * retrieve keys. By default, get will return the value for "key".
```

Comment:
that makes sense since go client get has an option field. for this func, we only have one arg, there is no option just "default".